cypress/cypress/integration/record.spec.ts

```diff
@@ -19,6 +19,7 @@ import {
   completeSubjectQuestion,
   updateMentorAnswer,
 } from "../support/helpers";
+import mentor from "../fixtures/mentor/clint_home";
```

Comment:
can you double check whether absolute imports work here? Prefer them if they do

Reply:
I think we could probably discard this pr since I redid the bash button in it's own PR and Aaron's suggestion page is going to take the place of this feature, but I can address the best practices changes in the bash-button PR?

Reply:
ok, sounds good. Once we merge your other PR we can close/delete this one